hbw/app/javascript/packs/hbw/components/form/services_table/services_table.tsx

```diff
@@ -345,6 +351,11 @@ const HBWFormServicesTable: React.FC<Props> = (props) => {
         searchQuery,
         onSearchQueryChange: handleSearchQueryChange
       }}
+      columnsSetting={{
+        columns:    columns.filter(column => column.isVisible),
+        allColumns: tableColumns,
```

Comment:
suggest some naming adjustment
```suggestion
      columnsSettings={{
        visibleColumns:    columns.filter(column => column.isVisible),
        columns: tableColumns,
```